cpp/arcticdb/python/python_module.cpp

```diff
@@ -227,12 +191,33 @@ void register_error_code_ecosystem(py::module& m) {
     setattr(m, "enum_value_to_prefix", enum_value_to_prefix);
     m.def("get_error_category", &get_error_category);
 
-    py::register_exception<InternalException>(m, "InternalException");
-    py::register_exception<NormalizationException>(m, "NormalizationException");
-    py::register_exception<MissingDataException>(m, "MissingDataException");
-    py::register_exception<SchemaException>(m, "SchemaException");
-    py::register_exception<StorageException>(m, "StorageException");
-    py::register_exception<SortingException>(m, "SortingException");
+    // legacy exception base type kept for backwards compat with Man Python client
+    struct ArcticCompatibilityException : public std::runtime_error {};
```

Comment:
This is _just_ a base exception type at the end of the day. Does calling it a `CompatibilityException` clarify much more than just calling it `ArcticBaseException` or something?

Reply:
See https://github.com/man-group/ArcticDB/pull/278#discussion_r1178147307

Reply:
I've now added a true base exception type too.